packages/@coorpacademy-components/src/organism/accordion/part/index.js

```diff
@@ -53,7 +73,9 @@ AccordionPart.propTypes = {
   title: PropTypes.string,
   content: PropTypes.node,
   onClick: PropTypes.func,
-  isOpen: PropTypes.bool
+  isOpen: PropTypes.bool,
+  moreIconType: PropTypes.string,
```

Comment:
I don't want to be able to change the icons

Reply:
I created a new component, which is accordion-cohort

Reply:
rollback Default